File: frontend/src/Domain/Dtos/SchoolDto.ts
import EntityBase from '@/domain/EntityBase.ts';
import AddressDto from '@/domain/dtos/AddressDto.ts';

interface SchoolDto extends EntityBase {
  name: string;
  email: string;
  phone: string;
  website: string;
  adminCount?: number;
  address: AddressDto;
}

export default SchoolDto;

File: Application/Dto/School/SchoolDto.cs
using Domain.Entities_v2.Types;

namespace Application.Dto.School;

public class SchoolDto: IBaseDto
{
    public int Id { get; set; }
    public DateTime Created { get; set; }
    public DateTime Updated { get; set; }
    public Status Status { get; set; }
    
    public string Name { get; set; } = null!;
    public string Email { get; set; } = null!;
    public string Phone { get; set; } = null!;
    public string Website { get; set; } = null!;
    
    public int? AdminCount { get; set; }
    
    public AddressDto Address { get; set; } = null!;
}